tests/wpt/metadata/MANIFEST.json

```diff
@@ -266,6 +266,10 @@
         "url": "/html-media-capture/capture_video_cancel-manual.html"
       },
       {
+        "path": "html/browsers/history/joint-session-history/joint-session-history-only-fully-active.html",
```

Comment:
Good catch, I think this comes from juggling several PRs at once, and having files leak from one branch to the other.